material_3_demo/lib/elevation_screen.dart

```diff
@@ -23,7 +23,10 @@ class ElevationScreen extends StatelessWidget {
               ),
             ),
           ),
-          ElevationGrid(surfaceTintColor: surfaceTint),
+          ElevationGrid(
+            surfaceTintColor: surfaceTint,
+            shadowColor: Colors.transparent,
```

Comment:
Just gave it another thought. I think setting it to transparent is a better way to only show the surface tint color. So LGTM! I can make the same change to the experimental one.

Reply:
@QuncCccccc yes I agree since the idea is to demonstrate normal shadow and only tint based elevation. That is how this looked before `Material` when using M3 was changed in Flutter. Originally this screen looked as I made, but a later change in the Framework broke the intent.